test/e2e/specs/test.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
     // automatically uses dev Server port from /config.index.js
     // default: http://localhost:8080
     // see nightwatch.conf.js
-    devServer = browser.globals.devServerURL
+    devServer = browser.globals.devServerURL // This link does not work for now
```

Comment:
If you rebase to the latest changes in master it will work. Please remove this comment